frontend/packages/dev-console/src/components/import/git/GitSection.tsx

```diff
@@ -46,10 +47,12 @@ const GitSection: React.FC = () => {
         />
       )}
       <ExpandCollapse
+        className="odc-expand-collapse"
         textExpanded="Hide Advanced Git Options"
         textCollapsed="Show Advanced Git Options"
       >
         <InputField
+          style={{ marginTop: '15px' }}
```

Comment:
We don't want to start supporting these kinds of customizations. The components should have the appropriate spacing. If we do need additional spacing we're better off creating a container to add such padding.